samples/highcharts/xaxis/minortickspermajor/demo.js

```diff
@@ -0,0 +1,10 @@
+Highcharts.chart('container', {
+    yAxis: {
+        minorTicks: true,
+        minorTicksPerMajor: 3
```

Comment:
This is technically correct, but can we use `minorTicksPerMajor: 2` instead? Since we now have minor ticks at 1/3 and 2/3 which is not a common real-live use case.